journal-temp/modernized/06-chapter-6.adoc

```diff
@@ -122,8 +130,8 @@ and since has not failed to stand by and help his faithful people in their times
 I proceed to give some account of my own share of suffering and exercise in those days,
 and of the manner thereof.
 
-A brief account of my commitment to Norwich castle, with other Friends,
-the 20th day of the Eleventh month, 1660.
+[.inline]
```

Comment:
Wasn't sure if this was the right thing to do, but it seemed like it needed to stand out some.